code/modules/mob/living/simple_animal/hostile/faithless.dm

```diff
@@ -34,11 +34,3 @@
 
 	do_footstep = TRUE
 	hardattacks = TRUE
-
-/mob/living/simple_animal/hostile/faithless/AttackingTarget()
-	. = ..()
-	if(. && prob(12) && iscarbon(target))
-		var/mob/living/carbon/C = target
-		C.Paralyze(60)
-		C.visible_message("<span class='danger'>\The [src] knocks down \the [C]!</span>", \
-				"<span class='userdanger'>\The [src] knocks you down!</span>")
```

Comment:
Why is this removed?

Reply:
I was in the code for it and missed it in a previous pr so i decided to hit it